### ISSUE TITLE
bool-compare: false positive when comparing to a non-`bool`

### ISSUE THREAD

**Issue by scop** [Antonboom/testifylint#23]
```go
	var foo any
	foo = true
	assert.Equal(true, foo)
```
...gets flagged with
```
bool-compare: use assert.True
```

Simply changing to `assert.True` does not work, as the "actual" is not a `bool`.

**Comment by Antonboom** [comment]
Thanks, `any` is ignored now
